src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -464,7 +465,7 @@
    * Builder for {@xsettingsClass.name}.
    */
   public static class Builder extends ClientSettings.Builder {
-    private final ImmutableList<UnaryCallSettings.Builder> unaryMethodSettingsBuilders;
+    private final ImmutableList<UnaryCallSettings.Builder<?, ?>> unaryMethodSettingsBuilders;
```

Comment:
Why aren't the UnaryCallSettings type pararmeters here aren't predictable?

Reply:
because this is a list of settings for all of the API methods, which each have different request and response types.